wagtail_wordpress_import/importers/wordpress.py

```diff
@@ -329,6 +343,26 @@ def connect_page_categories(self, page, category_model, item):
 
             page.categories = page_categories
 
+    def reparent_pages(self, imported_pages):
+        """
+        Reparent any pages that have a wp_post_parent and update their slug
+        """
+        for page in imported_pages:
+            wp_post_parent_id = page.wp_post_meta.get("wp_post_parent", 0)
+            original_slug = page.wp_post_meta.get("original_slug", None)
+
+            # If this page has a parent and still has an original slug then reparent it
+            if wp_post_parent_id != 0 and original_slug is not None:
+                wp_post_parent_page = page.__class__.objects.get(wp_post_id = wp_post_parent_id)
+                # Check that the parent isn't the same already
+                if page.get_parent() != wp_post_parent_page:
+                    page.move(wp_post_parent_page, pos="last-child")
+                    page = page.specific_class.objects.get(pk=page.pk)
+                    # Restore the correct slug
+                    page.slug = original_slug
```

Comment:
note that if there are pages and posts under a wp_post_parent there could be issues as we don't consider the slug clash resolution in #175